nameko_stripe/constants.py

```diff
@@ -0,0 +1,11 @@
+CONFIG_KEY = "STRIPE"
+
+DEFAULT_EVENT_PATH = "/stripe/events/"
+
+APP_NAME = "nameko-stripe"
+
+APP_URL = "https://github.com/marcuspen/nameko-stripe"
+
+VERSION = "0.1.3"  # TODO Find better place
```

Comment:
The common problem of adding `__verison__` into `__init__.py` of the main package is that running `setup.py` in an environment without all dependencies installed would fail if it contains imports requiring them (which it does in our case).

I quite like option 4 of the following list:
https://packaging.python.org/guides/single-sourcing-package-version/#single-sourcing-the-version

Reply:
It makes sense.

Option 1 is quite common though, but I agree that option 4 looks quite agnostic. The less ugly I'd say :)

Considering the amount of extra effort and the current size of this PR, I would suggest that we attack this in another PR. Does it sound okay?